controllers/subscription/csvlocator/csvlocator.go

```diff
@@ -67,44 +71,87 @@ func (l *ConfigMapCSVLocator) GetCSV(ctx context.Context, client k8sclient.Clien
 
 		// Get the ConfigMap
 		csvConfigMap := &corev1.ConfigMap{}
-		if err := client.Get(ctx, k8sclient.ObjectKey{
+		if err = client.Get(ctx, k8sclient.ObjectKey{
 			Name:      ref.Name,
 			Namespace: ref.Namespace,
 		}, csvConfigMap); err != nil {
 			return csv, fmt.Errorf("error retrieving ConfigMap %s/%s: %v", ref.Namespace, ref.Name, err)
 		}
 
 		// The ConfigMap may contain other manifests other than the CSV. Iterate
-		// through the content and skip the ones that have a kind other than
+		// through the data and skip the ones that have a kind other than
 		// ClusterSeerviceVersion
```

Comment:
Can we add a note that this is required for openshift 4.8 and put a similar note for the 4.9 block

Reply:
done